experimental/generation/generator/packages/library/templates/professional/config.json.lg

```diff
@@ -11,7 +11,7 @@ ${join(foreach(locales, locale, localeTemplate(locale)), ','+EOL())}
 ```
 
 # localeTemplate(locale)
--```    "./${locale}/${prefix}.${locale}.lu": {
+-```    "./${knowledgeDir()}${locale}/${prefix}.${locale}.lu": {
```

Comment:
For the latest format for cross-train, ./${knowledgeDir()}${locale}/${prefix}.${locale}.lu should be changed to ${prefix}.${locale}

Reply:
I'm not sure how this fits in with composer, so I'm leaving it right now.